CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Change Log
 
+## [0.17.0](https://github.com/redhat-developer/vscode-xml/milestone/21?closed=1) (June 24, 2021)
+
+### Enhancements
+ * Added `closingBracketNewLine` formatting option to package.json. See [#508](https://github.com/redhat-developer/vscode-xml/pull/508).
+ * Support workspace trust. See [#506](https://github.com/redhat-developer/vscode-xml/issues/506).
+ * Optimize images size. See [#497](https://github.com/redhat-developer/vscode-xml/pull/497).
+ * Better handle for failed binary downloads due to "407 Proxy Authentication Required". See [#492](https://github.com/redhat-developer/vscode-xml/issues/492).
+ * Update to vscode language client 7. See [#478](https://github.com/redhat-developer/vscode-xml/issues/478).
+ * Bind XSD, DTD with CodeLens. See [#395](https://github.com/redhat-developer/vscode-xml/issues/395).
+ * Added Auto Rename Tag. See [#130](https://github.com/redhat-developer/vscode-xml/issues/130).
+ * Promote features of `vscode-xml` in documentation. See [#475](https://github.com/redhat-developer/vscode-xml/issues/475).
+ * Upgrade `@redhat-developer/vscode-redhat-telemetry` to 0.1.1. See [#504](https://github.com/redhat-developer/vscode-xml/issues/504).
+ * Generate CodeLens to associate a grammar/schema. See [#1049](https://github.com/eclipse/lemminx/pull/1049).
+ * Support for `textDocument/selectionRange`. See [#1021](https://github.com/eclipse/lemminx/issues/1021).
```

Comment:
If you reference an issue/PR from lemminx, change the link text to "eclipse/lemminx#1021"